staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/tlsconfig.go

```diff
@@ -152,7 +151,7 @@ func (c *DynamicServingCertificateController) syncCerts() error {
 		newClientCAPool := x509.NewCertPool()
 		newClientCAs, err := cert.ParseCertsPEM(newContent.clientCA.caBundle)
 		if err != nil {
-			return fmt.Errorf("unable to load client CA file: %v", err)
+			return fmt.Errorf("unable to load client CA file %q: %v", string(newContent.clientCA.caBundle), err)
```

Comment:
since `caBundle` could hold many certificates this error message could become quite large , wdyt?

Reply:
agree... what's the purpose of echoing the bundle content?

Reply:
Hmm, this makes some sense to me; if eg its an empty string, or has a bunch of null bytes or something, this would be a useful way to see that. Maybe if your cert bundle changes regularly, this log line would be the only way to know what went wrong